src/Libraries/Microsoft.Extensions.AI.Evaluation.Reporting/CSharp/ReportingConfiguration.cs

```diff
@@ -162,6 +163,7 @@ public ReportingConfiguration(
     /// A new <see cref="ScenarioRun"/> with the specified <paramref name="scenarioName"/> and
     /// <paramref name="iterationName"/>.
     /// </returns>
+    /// <related type="Article" href="/dotnet/ai/tutorials/evaluate-with-reporting">Tutorial: Evaluate a model's response with response caching and reporting</related>
```

Comment:
Perhaps it may be better to add this on the interface IEvaluator (otherwise we should probably add it to all the other evaluators like FluencyEvaluator, RTCEvaluator etc. in this project)

Reply:
Looks like the link is present on IEvaluator which is great - so then the question is just whether CoherenceEvaluator needs to be special v/s whether we should also add it to the other evaluators I mentioned above.

Reply:
CoherenceEvaluator is the only evaluator that's used in [the quickstart](https://learn.microsoft.com/en-us/dotnet/ai/quickstarts/evaluate-ai-response). The RTC evaluator is used in the tutorial, so I'll add a link to that from the RTC evaluator.